ItemLevelDownloader/Engine/artifactEngine.ts

```diff
@@ -62,14 +65,16 @@ export class ArtifactEngine {
             retryCount = retryCount ? retryCount : 0;
             if (item.itemType === models.ItemType.File) {
                 if (minimatch(item.path, artifactEngineOptions.itemPattern, { dot: true })) {
-                    console.log("Processing '%s'", item.path);
+                    this.logger.logInfo("Processing " + item.path);
                     const contentStream = await sourceProvider.getArtifactItem(item);
-                    console.log("got read stream for item: " + item.path);
+                    this.logger.logInfo("Got download stream for item: " + item.path);
```

Comment:
this can be download or upload, right? should it be read stream then?

Reply:
this will be download, putArtifactItem will be upload.